Source/Krypton Components/Krypton.Toolkit/Controls Visuals/Notification Toast/Basic/VisualToastNotificationBasicForm.cs

```diff
@@ -0,0 +1,235 @@
+                    break;
+                case KryptonToastNotificationIcon.SystemAsterisk:
+                    break;
+                case KryptonToastNotificationIcon.Stop:
+                    SetIcon(ToastNotificationImageResources.Toast_Notification_Stop_128_x_128);
+                    break;
+                case KryptonToastNotificationIcon.Error:
+                    SetIcon(ToastNotificationImageResources.Toast_Notification_Critical_128_x_128);
+                    break;
+                case KryptonToastNotificationIcon.Warning:
+                    SetIcon(ToastNotificationImageResources.Toast_Notification_Warning_128_x_115);
+                    break;
+                case KryptonToastNotificationIcon.Information:
+                    SetIcon(ToastNotificationImageResources.Toast_Notification_Information_128_x_128);
+                    break;
+                case KryptonToastNotificationIcon.Shield:
+                    if (OSUtilities.IsWindowsEleven)
+                    {
+                        SetIcon(ToastNotificationImageResources.Toast_Notification_UAC_Shield_Windows_11_128_x_128);
+                    }
+                    else if (OSUtilities.IsWindowsTen)
+                    {
+                        SetIcon(ToastNotificationImageResources.Toast_Notification_UAC_Shield_Windows_10_128_x_128);
+                    }
+                    else
+                    {
+                        SetIcon(ToastNotificationImageResources.Toast_Notification_UAC_Shield_Windows_7_and_8_128_x_128);
+                    }
+                    break;
+                case KryptonToastNotificationIcon.WindowsLogo:
+                    break;
+                case KryptonToastNotificationIcon.Application:
+                    break;
+                case KryptonToastNotificationIcon.SystemApplication:
+                    break;
+                case KryptonToastNotificationIcon.Ok:
+                    SetIcon(ToastNotificationImageResources.Toast_Notification_Ok_128_x_128);
+                    break;
+                case KryptonToastNotificationIcon.Custom:
+                    break;
+                case null:
+                    SetIcon(null);
+                    break;
+                default:
+                    throw new ArgumentOutOfRangeException();
+            }
+        }
+
+        private void SetIcon(Bitmap? image) => pbxIcon.Image = image;
+
+        private void UpdateLocation()
+        {
+            //Once loaded, position the form to the bottom left of the screen with added padding
+            Location = new Point(Screen.PrimaryScreen.WorkingArea.Width - Width - 5,
+                Screen.PrimaryScreen.WorkingArea.Height - Height - 5);
+        }
+
+        private void VisualToastNotificationBasicForm_Load(object sender, EventArgs e)
+        {
+            UpdateLocation();
+
+            ShowCloseButton();
+
+            _timer?.Start();
```

Comment:
When will the timer be null?
Remove nullable

Reply:
Removed nullable